source/renderer/app/components/status/DaedalusDiagnostics.js

```diff
@@ -568,6 +569,15 @@ export default class DaedalusDiagnostics extends Component<Props, State> {
 
     return (
       <div className={styles.component}>
+        <DialogCloseButton
+          className={styles.closeButton}
+          icon={closeCrossThin}
+          onClose={onClose}
+        />
+        {/* <button className={styles.closeButton} onClick={() => onClose()}>
```

Comment:
@daniloprates why do we need to keep this commented out part?

Reply:
@nikolaglumac done!